Compile bug: How to build Llama.cpp for ARM64 Windows with MSVC?

### Git commit

latest version.

### Operating systems

Windows

### GGML backends

CPU

### Problem description & steps to reproduce

https://github.com/ggml-org/llama.cpp/blob/6c442f42ff25564a0cd6b1435d9abc1b0178eac5/docs/build.md#cpu-build

From above link, it mentioned we can compile it for ARM64 windows with 'build-arm64-windows-MSVC', but when I compile it got the error info below. I remember last year we could successfully compile it through MSVC.
The reason why I want to compile it through MSVC is that I hope to enable KleidiAI to improve the performance on Windows On ARM device. 

### First Bad Commit

_No response_

### Compile command

```shell
cmake -B build --preset build-arm64-windows-MSVC -D GGML_OPENMP=OFF -DGGML_CPU_KLEIDIAI=OFF -DLLAMA_CURL=OFF
```

### Relevant log output

```shell
cmake -B build --preset build-arm64-windows-MSVC -D GGML_OPENMP=OFF -DGGML_CPU_KLEIDIAI=OFF -DLLAMA_CURL=OFF
CMake Error: No such preset in C:/Source/Projects/llama.cpp: "build-arm64-windows-MSVC"
Available configure presets:

  "x64-linux-gcc-debug"
  "x64-linux-gcc-release"
  "x64-linux-gcc-reldbg"
  "x64-linux-gcc+static-release"
  "arm64-windows-llvm-debug"
  "arm64-windows-llvm-release"
  "arm64-windows-llvm+static-release"
  "arm64-apple-clang-debug"
  "arm64-apple-clang-release"
  "arm64-apple-clang+static-release"
  "x64-windows-llvm-debug"
  "x64-windows-llvm-release"
  "x64-windows-llvm-reldbg"
  "x64-windows-llvm+static-release"
  "x64-windows-msvc-debug"
  "x64-windows-msvc-release"
  "x64-windows-msvc+static-release"
  "x64-windows-sycl-debug"
  "x64-windows-sycl-debug-f16"
  "x64-windows-sycl-release"
  "x64-windows-sycl-release-f16"
  "x64-windows-vulkan-debug"
  "x64-windows-vulkan-release"
```

--- a/docs/build.md
+++ b/docs/build.md
@@ -59,8 +59,6 @@ cmake --build build --config Release
     cmake --preset arm64-windows-llvm-release -D GGML_OPENMP=OFF
     cmake --build build-arm64-windows-llvm-release
     ```
-    Building for arm64 can also be done with the MSVC compiler with the build-arm64-windows-MSVC preset, or the standard CMake build instructions. However, note that the MSVC compiler does not support inline ARM assembly code, used e.g. for the accelerated Q4_0_N_M CPU kernels.
-
     For building with ninja generator and clang compiler as default:
       -set path:set LIB=C:\Program Files (x86)\Windows Kits\10\Lib\10.0.22621.0\um\x64;C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.41.34120\lib\x64\uwp;C:\Program Files (x86)\Windows Kits\10\Lib\10.0.22621.0\ucrt\x64
       ```bash

